lighty-examples/lighty-bgp-community-restconf-app/LIGHTY-BGP.postman_collection.json

```diff
@@ -66,18 +71,21 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "<topology xmlns=\"urn:TBD:params:xml:ns:yang:network-topology\">\n    <topology-id>bgp-example-ipv4-topology</topology-id>\n    <topology-types>\n        <bgp-ipv4-reachability-topology xmlns=\"urn:opendaylight:params:xml:ns:yang:odl-bgp-topology-types\"></bgp-ipv4-reachability-topology>\n    </topology-types>\n    <rib-id xmlns=\"urn:opendaylight:params:xml:ns:yang:odl-bgp-topology-config\">bgp-example</rib-id>\n</topology>"
+							"raw": ""
 						},
 						"url": {
-							"raw": "{{lighty-host}}/restconf/data/network-topology:network-topology/topology=bgp-example-ipv4-topology",
```

Comment:
I would stay with parameter {lighty-host} - because my lighty is not running on localhost when I invoke the requests :)